### ISSUE TITLE
[BUG] Color Contrast failure in search bar (dark mode only)

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-windows#1149]
**Describe the bug**
In dark mode, the text in the properties search pane is always black, so it fails contrast requirements against the dark background.

**To Reproduce**
1. Run AIWin in dark mode
2. Display an element's properties in live inspect mode
3. Set focus to the properties search edit
4. Start to type
5. Look at the text color

**Expected behavior**
Text color should satisfy contrast requirements against the background of the edit control

**Actual behavior**
Text color does not satisfy contrast requirements

**Screenshots or .GIF**
![search-text-color](https://user-images.githubusercontent.com/45672944/125830846-ba1a6506-979e-4ad8-b41a-0e89e5c95184.gif)

**Desktop (please complete the following information):**
 - OS: Windos 10 21H1
 - Accessibility Insights for Windows Version: 1.1.1656.1

**Additional context**
Priority requested -

**Comment by DaveTryon** [comment]
Confirmed fixed in 1.1.1657.1
